support/mock-data/Search/cgov/es/all/tumor.json

```diff
@@ -122,4 +122,4 @@
 		}
```

Comment:
Doubt this file is needed if it doesn't have the same naming pattern for Search mock files.